app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/statistikk/StatistikkModel.kt

```diff
@@ -296,20 +314,53 @@ class StatistikkModel(
                 }
             }
             is HardDelete -> {
-                // noop
+                if (hendelse.grupperingsid != null && hendelse.merkelapp != null) {
```

Comment:
jeg har ikke fått med meg at vi skal legge på statistikk for harddelete. Men hvis det er det dere har kommet frem til ser dette ok ut for meg.
Blir kanskje litt smør på flesk siden vi allerede har statistikk på dette i metabase. Forskjellen vil kanskje være at dette oppdateres fortløpende, mens metabase er en daglig eksport P.T.